### PR TITLE
Add DOMEI

### PR DESCRIPTION
**Pull request by MaygoDev**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments

Added [DOMEI](https://domei.eu/) server on the repo.

### CODE REVIEW

File: minecraft_servers/domei/manifest.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  "nice_name": "DOMEI",
+  "direct_ip": "domei.eu",
+  "server_wildcards": [
+    "%.domei.eu"
+  ],
+  "supported_languages": [
+     "fr"
+  ],
+  "social": {
+    "web": "https://domei.eu/",
```
Comment (JakobBruening) [inline minecraft_servers/domei/manifest.json:12]: 1. Website is down (Error 521)
2. Remove "/" at the end

File: minecraft_servers/domei/manifest.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    "web_shop": "https://domei.eu/boutique/",
```
Comment (JakobBruening) [inline minecraft_servers/domei/manifest.json:13]: Remove "/" at the end

File: minecraft_servers/domei/manifest.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      "color": "#55FF55"
+    },
+    "ssb": {
+      "name": "Super Smash Bows",
+      "color": "#FFFF55"
+    }
+  },
+  "discord": {
+    "server_id": 837316390708445204,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/domei/manifest.json:58]: Please use "false" as it is the default value. "true" is currently only available for partners.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/domei/manifest.json (hunk 3 above)
Comment (MaygoDev) [reply minecraft_servers/domei/manifest.json:58]: Haven't see this, thanks!

File: minecraft_servers/domei/manifest.json (hunk 1 above)
Comment (MaygoDev) [reply minecraft_servers/domei/manifest.json]: For us, the website hasn't any problems. Tell us if it still appear offline on your side! :)

**Review by JakobBruening — APPROVED** [review]
Thank you!

**Pull request merged**